styles/mint.json

```diff
@@ -0,0 +1,115 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 2,
+	"title": "Mint ",
+	"settings": {
+		"color": {
+			"palette": [
+				{
+					"color": "#e4efeb",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#f1fefb",
+					"name": "Base / Two",
+					"slug": "base-2"
+				},
+				{
+					"color": "#00000025",
+					"name": "Base / Three",
+					"slug": "base-3"
+				},
+				{
+					"color": "#000000",
+					"name": "Contrast",
+					"slug": "contrast"
+				},
+				{
+					"color": "#353535",
+					"name": "Contrast / Two",
+					"slug": "contrast-2"
+				},
+				{
+					"color": "#A4A4A4",
+					"name": "Contrast / Three",
+					"slug": "contrast-3"
+				}
+			]
+		}
+	},
+	"styles": {
+		"blocks": {
+			"core/navigation": {
+				"typography": {
+					"fontStyle": "normal",
+					"fontWeight": "400"
+				}
+			},
+			"core/pullquote": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--instrument-sans)",
+					"fontSize": "var(--wp--preset--font-size--x-large)",
+					"fontStyle": "normal",
+					"fontWeight": "600",
+					"lineHeight": "1.3"
+				}
+			},
+			"core/quote": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--instrument-sans)",
+					"fontSize": "var(--wp--preset--font-size--large)",
+					"fontStyle": "normal",
+					"fontWeight": "400"
+				}
+			},
+			"core/site-title": {
+				"typography": {
+					"fontFamily": "var(--wp--preset--font-family--instrument-sans)"
```

Comment:
if we do this this way it won't work with the rest of the patterns that are using heading/body naming for their fonts. Why are we defining the font on the global theme.json if it's not using it? We should do it on the variation instead. Core will be adding a way for users to add new fonts soon so the theme doesn't need to worry about providing the user with extras

Reply:
I've compared it with TT3 where all fonts were put in the theme.json. Haven't thought about the heading/body naming. Just added an update.